deep-learning/src/main/python/synapse/ml/dl/DeepVisionModel.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+import numpy as np
+import torch
+import torchvision.transforms as transforms
+from horovod.spark.lightning import TorchModel
+from PIL import Image
+from synapse.ml.dl.PredictionParams import PredictionParams
+from pyspark.ml.param import Param, Params, TypeConverters
+from pyspark.sql.functions import col, udf
+from pyspark.sql.types import DoubleType
+from synapse.ml.dl.utils import keywords_catch
+
+
+class DeepVisionModel(TorchModel, PredictionParams):
+
+    transform_fn = Param(
+        Params._dummy(),
+        "transform_fn",
+        "A composition of transforms used to transform and augnment the input image, should be of type torchvision.transforms.Compose",
+    )
+
+    @keywords_catch
+    def __init__(
+        self,
+        history=None,
+        model=None,
+        input_shapes=None,
+        optimizer=None,
+        run_id=None,
+        _metadata=None,
+        loss=None,
+        loss_constructors=None,
+        # diff from horovod
+        transform_fn=None,
+        label_col="label",
+        image_col="image",
+        prediction_col="prediction",
```

Comment:
If we dont use the actual predictor API in SparkML will we work OOTB with the Evaluators?